uiautomationutilities/src/main/java/com/microsoft/identity/client/ui/automation/rules/InstallBrokerTestRule.java

```diff
@@ -52,6 +54,13 @@ public Statement apply(final Statement base, final Description description) {
             public void evaluate() throws Throwable {
                 Logger.i(TAG, "Applying rule....");
                 Logger.i(TAG, "Installing broker: " + ((App) broker).getAppName());
+
+                if (BuildConfig.PRE_INSTALL_LTW) {
```

Comment:
could you please elaborate more/ add comments on what this is needed for?

If this is to cover "install LTW first and everything else must not break", we need to support both .. (where LTW is, or isn't installed).

Not sure how that would be executed. Execute the suite twice with buildconfig flag set and unset?

Reply:
Yes, discussed this with Sowmya and most appropriate in my opinion is to have this flag be set based on a pipeline parameter/variable. That way you can choose to run this suite with the flag or not. you can run it twice to cover "Install LTW first and everything else must not break"